.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Create a feature request to help us achieve the best solutions
+title: "[FEATURE]"
+labels: feature
+assignees: ''
+
+---
+
+**Describe how this feature will help us improve**
+Describe why do you think this feature is needed. 
+
+**Is your proposal related to a problem?**
+Describe the issue or type its number (eg. #223)
+
+**Describe the solution you'd considered**
+A concise description of your preferred solution. Things to address include:
+
+  - Details of the technical implementation
+  - Tradeoffs made in design decisions
+  - Caveats and considerations for the future
+
+If there are multiple solutions, please present each one separately. Save comparisons for the very end.
+
+**Additional context**
+Anything you would like to add but was not sure which section it fits.
```

Comment:
I think we don't need this section. Instead, it would be nice to have a checklist fo acceptance criteria for a task